nss-capricious-finches/nutshell
# create user on sign up module

# NEW CREATE USER MODULE
----------------------------------

# Description of Proposed Changes
Steps to Test
Outline the steps to test
1.Fill out the sign up form with a unique user name and check your api and session storage to confirm that the user input has been objectified and added to each.
2. Next fill out the sign up form with a user name that already exist. If so, it should not be added to the API nor Session Storage, along with an alert.

```
git fetch --all
git checkout dillan
```

# Impacted Areas in Application
newUserSignUp.js
